spec/authy/phone_verification_spec.rb

```diff
@@ -16,6 +16,20 @@
       expect(response.message).to eq "Text message sent to +1 111-111-1111."
     end
 
+    it "should send the code via SMS with code length" do
+      pending("API is not returning expected response in this case. The test phone number is invalid.")
+      response = Authy::PhoneVerification.start(
+        via: "sms",
+        country_code: "1",
+        phone_number: "111-111-1111",
+        code_length: "2387"
```

Comment:
@Tolsee accordingly with the documentation: `code_legth` Allowed values are 4-10, so this test is using an invalid value for code length.

also, in order to avoid adding a pending test, please change the phone number to a valid number eg. +1 2015550123, this is an [example number](https://github.com/googlei18n/libphonenumber/blob/master/resources/PhoneNumberMetadata.xml#L26159) from libphonenumber, so you can use it and it is valid for US

Reply:
It will be best If we can change all the pending test with a valid phone number. Do you want a PR on that? I can work on that If you want to.

Reply:
If you can do it, it will be great, thanks in advance

Reply:
I will merge this PR, you can create another one with the test fixes.